packages/atlas/src/views/studio/YppDashboard/YppDashboard.styles.ts

```diff
@@ -81,3 +82,41 @@ export const StyledSvgAlertsInformative24 = styled(SvgAlertsInformative24)`
     fill: ${cVar('colorTextStrong')};
   }
 `
+export const StyledTab = styled(Tabs)`
+  margin-bottom: 24px;
+`
+
+export const SettingsInputsWrapper = styled.div`
+  margin: 32px 0;
```

Comment:
same here

Reply:
Also, the gap between tabs and the wrapper should be 56px for smaller sizes as well.